### DISCUSSION TITLE
How do i remove some data from a list rather than deleting the whole model itself

### DISCUSSION THREAD

**Discussion by UndiedGamer** [prisma/prisma#9214] [Q&A] [answered]
List has a push method to add something to a list but is there a method to remove something from a list?

**Comment by ryands17** [comment, marked as answer]
@UndiedGamer 👋 
Currently you would need to fetch the data for the scalar list, delete and then insert again. We have a request for making this easier here so it would be great if you could add a 👍 to this: https://github.com/prisma/prisma/issues/7401

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
